spec/javascript/utils/compiler/mixins.test.js

```diff
@@ -215,6 +215,36 @@ describe("mixins.js", () => {
       expect(result.fullMixin).toEqual(joinedItems)
       expect(disregardWhitespace(result.replaceWith)).toEqual(disregardWhitespace("Global.slot = 1;"))
     })
+
+    it("Should replace Mixin variables with parameter objects if it's given as the only param", () => {
```

Comment:
I think maybe this could be a warning on the linter?

Reply:
Hmm maybe, but it is intended, not a warning. It's difficult to convey though